gender_analysis/gender.py

```diff
@@ -198,3 +190,18 @@ def uses(self, identifier):
             self.uses_name(identifier) or
             self.uses_pronoun(identifier)
         )
+
+    def associations(self):
```

Comment:
Can we rename this method? I'd also make it a property. Trying to avoid the word `associated` because I think that's going to get confusing, seeing as a lot of the analytic functions in this package involve making associations between gender and _other_ words.